draft-xyz-rats-psa-endorsements.md

```diff
@@ -131,94 +140,149 @@ firmware measurement associated with Implementation ID
-with a `comid-updates` relation, whereas an increase in the PATCH indicator
-would have a corresponding `comid-patches` relation between the involved
-components.  Note that `comid-updates` relations would only occur between
-firmware RoT descriptors that have PATCH number 0 and "adjacent" MINOR numbers
-as illustrated in {{fig-updates-patches}}.  Note that changing the MAJOR
-version number would typically result in a separate product / upgrade chain.
+{: #sec-identity}
+
+Identity Claims carry the verification key associated with the Initial
+Attestation Key (IAK) of a PSA device.
+
+Each verification key is encoded alongside the corresponding device Instance ID
+in a CoMID identity claim.
+
+The Instance ID is encoded using `tagged-ueid` as `$device-id-type`.
+
+The IAK public key is encoded as a COSE Key according to Section 7 of
+{{!RFC8152}} and wrapped in the `COSE_KeySet`. The number of items in the
+`COSE_KeySet` MUST be 1.
+
+The example in {{ex-identity-claim}} shows a CoMID identity claim carrying a
+secp256r1 EC public IAK associated with Instance ID `4ca3...d296`.
 
-~~~ goat
-{::include art/updates-patches.txt}
 ~~~
-{: #fig-updates-patches title="Updates, Patches Relations and Semantic Versioning"}
+{::include examples/instance-pub.diag}
+~~~
+{: #ex-identity-claim title="Example Identity Claim"}
 
-The Reference Value CoMID of the patching or updating firmware has a
-`linked-tag-entry` map populated as follows:
+### CoMID Layout
 
-* key 0 contains the tag identifier of the Reference Value CoMID of the patched
-  or updated firmware;
-* key 1 contains one of `comid-patches` or `comid-updates` relations.
+An Identity CoMID can carry as many Identity Claims as needed.
 
-{{ex-linked-tag}} provides an example of a Reference Value CoMID patching
-another Reference Value CoMID with a UUID tag identifier
-`3f06af63-a93c-11e4-9797-00505690773f`.
+## Certification
+
+{: #sec-certificates}
+
+A PSA Certification certify a PSA device or a PSA hardware or software RoT, within a PSA device.
+PSA Certification claim contains the metadata about the certificate. The claim is 
+expressed by creating a new CoMID Tag for provisioning certification details.
+
+Within a CoMID tag, certification claim is encoded in a CoMID endorsement map.  Specifically:
+
+* The Implementation ID is encoded using the module type (2) entry in the
+  `element-name` map with a `tagged-impl-id` type;
+
+* The metadata associated with the certification is encoded using the extension
+  in endorsed-value-map.`cert-meta` structure extends the `endorsed-value` map and
+  carries various attributes of a certificate.
+  
+* The CoMID linked tags contain one or more entries of linked-tag-map, each entry carries
+  tag id of CoMID, which it certifies and tag-rel type set to `comid.supplements` relation, 
+  as illustrated in {{fig-cert-link}}.
+
+The sematics of Provisioning certificates is illustrated in {{fig-cert-ext}}.
```

Comment:
strictly speaking that's just an example, the semantics should be provided somewhere else (e.g., expanding the bullet list below).

I think we should: first provide the CDDL structure, secondly the semantics of the fields, and finally a working example.